test/val/val_id_test.cpp

```diff
@@ -148,6 +153,7 @@ TEST_F(ValidateIdWithMessage, OpLineFileBad) {
 %4 = OpVariable %3 Input)";
   CompileSuccessfully(spirv.c_str());
   EXPECT_EQ(SPV_ERROR_INVALID_ID, ValidateInstructions());
+  EXPECT_THAT(getDiagnosticString(), HasSubstr("ID 1 has not been defined"));
```

Comment:
👍  good catch. I had actually read every single error message to make sure we're getting the correct message. Looks like I had missed this one.

(ID 1 is in fact %2... that's why the error is complaining that ID 1 is not defined).